src/Adapter/Image/Uploader/AbstractImageUploader.php

```diff
@@ -1,12 +1,11 @@
 <?php
 /**
- * Copyright since 2007 PrestaShop SA and Contributors
- * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ * 2007-2020 PrestaShop SA and Contributors
  *
  * NOTICE OF LICENSE
  *
  * This source file is subject to the Open Software License (OSL 3.0)
- * that is bundled with this package in the file LICENSE.md.
+ * that is bundled with this package in the file LICENSE.txt.
```

Comment:
This is the older header Usually I don't care but the initial one being correct it's too bad to modify it into an invalid one ^^

Reply:
not sure how it happened, I didn't modify it intentionally :cold_sweat:

Reply:
Probably your IDE or something like that 🤷‍♂️